app/assets/stylesheets/_settings.scss

```diff
@@ -81,9 +81,9 @@ $print-transparent-backgrounds: true;
 
 $breakpoints: (
```

Comment:
I would be inclined to keep the foundation settings file as is. So that that they match across all projects that are using foundation.